EU4toV2/Data_Files/blankMod/output/history/pops/1836.1.1/Burma.txt

```diff
@@ -1024,11 +1024,11 @@
 		size = 5075
 	}
 
-	bureaucrats = {
-		culture = british
-		religion = protestant
-		size = 25
-	}
+	#bureaucrats = {
```

Comment:
One more set of things - commenting out the pop doesn't work well, either, because then the converter won't properly read it for setting pop totals in the province.

Reply:
Fixed